web_widget_char_size/static/src/js/char_widget.esm.js

```diff
@@ -0,0 +1,10 @@
+/** @odoo-module **/
+
+import {CharField} from "@web/views/fields/char/char_field";
+
+const _extractProps = CharField.extractProps;
+CharField.extractProps = ({attrs, field}) => {
+    return Object.assign(_extractProps({attrs, field}), {
+        maxLength: field.size || attrs.options.size,
```

Comment:
nitpick(non-blocking): According to https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength it should be without the capital 'L'

Still works anyways tho

Reply:
Required to update actual definition in order to be aligned with HTML attributes name:
https://github.com/odoo/odoo/blob/16.0/addons/web/static/src/views/fields/char/char_field.js#L80

Reply:
Everything is clear now, it's good as it is, thanks!